oneview_redfish_toolkit/tests/test_util.py

```diff
@@ -188,8 +188,9 @@ def test_load_registries_valid_registry_dir_valid_dict(self):
         except Exception as e:
             self.fail('Failed to load registries files: {}'.format(e.msg))
 
+    @mock.patch.object(util, 'check_oneview_availability')
     @mock.patch.object(util, 'OneViewClient')
```

Comment:
Do we still need this OneViewClient mock?

Reply:
No, once load_config() does not instantiate it anymore. Removing it.